### PR TITLE
Release notes for Percona Server 5.6.30-76.3

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.30-76.3 on May 25th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.30-76.3/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.30 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-30.html>`_, including all the bug fixes in it, |Percona Server| 5.6.30-76.3 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.30-76.3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.30-76.3>`_.
+
+Bugs Fixed
+==========
+ 
+ When :ref:`tokudb_read_free_replication` was enabled for |TokuDB| and there was no explicit primary key for the replicated |TokuDB| table there could be duplicated records in the table on update operation. Bug fixed :bug:`1536663` (:tokubug:`950`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst]: Need the details of the fix

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+ Attempting to execute a non-existing prepared statement with :ref:`response_time_distribution` plugin enabled could lead to a server crash. Bug fixed :bug:`1538019`.
+
+ |TokuDB| was using using different memory allocators this was causing ``safemalloc`` warnings in debug builds and crashes because memory accounting didn't add up. Bug fixed :bug:`1546538` (:tokubug:`962`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst]: Needs rephrasing, @georgelorchpercona

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+ Buffer pool may fail to remove dirty pages for a particular tablespace from the flush list. Bug fixed :bug:`1552673`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst]: other bugs

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+ Fixed heap allocator/deallocator mismatch in :ref:`scalability_metrics_plugin`. Bug fixed :bug:`1581051`.
+
+ |Percona Server| is now built with system ``zlib`` library instead of the older bundled one. Bug fixed :bug:`1108016`.
+
+ Fixed memory overhead per page in the |InnoDB| buffer pool. The fix was ported from Facebook patch `#91e979e <https://github.com/facebook/mysql-5.6/commit/91e979e8436b83400e918fa0f251036e50d0cb5f>`_. Bug fixed :bug:`1536693` (upstream :mysqlbug:`72466`).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst]: s/Fixed/Reduced
The fix was not ported exactly, but merely based on the Facebook version

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.30-76.3 on May 25th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.30-76.3/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.30 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-30.html>`_, including all the bug fixes in it, |Percona Server| 5.6.30-76.3 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.30-76.3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.30-76.3>`_.
+
+Bugs Fixed
+==========
+ 
+ When :ref:`tokudb_read_free_replication` was enabled for |TokuDB| and there was no explicit primary key for the replicated |TokuDB| table there could be duplicated records in the table on update operation. The fix disables :ref:`tokudb_read_free_replication` for tables without explicit primary key and does rows lookup for ``UPDATE`` and ``DELETE`` binary log events and issues warning. Bug fixed :bug:`1536663` (:tokubug:`950`).
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst:14]: What does tokudb_read_free_replication refer to here? There isn't a specific option with this name. RFR is enabled with two different options, tokudb_rpl_lookup_rows and tokudb_rpl_unique_checks.

**Comment by george-lorch** [comment]
Other than my one question/comment, LGTM

File: doc/source/release-notes/Percona-Server-5.6.30-76.3.rst (hunk 5 above)
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst:14]: It links to https://www.percona.com/doc/percona-server/5.6/tokudb/using_tokudb.html#read-free-replication

Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.30-76.3.rst:14]: Ack, looks good...

**Pull request merged**